geometry_msgs/Pose query
---
nav_msgs/Path[] safe_paths